packages/tldraw/src/lib/Tldraw.tsx

```diff
@@ -123,6 +126,9 @@ export function Tldraw(props: TldrawProps) {
 	if (preloadingError) {
 		return <ErrorScreen>Could not load assets. Please refresh the page.</ErrorScreen>
 	}
+
+	const embedsWithDefaults = embeds ?? DEFAULT_EMBED_DEFINITIONS
```

Comment:
also this name `embedsWithDefaults` is not really accurate since we're not merging in the defaults like with the tools and shapes (this is good IMO, if a little inconsistent in terms of API design).


but yeah anyway I'd keep it named `embeds`

```ts
const embeds = useShallowArrayIdentity(rest.embeds ?? DEFAULT_EMBED_DEFINITIONS)
```